releasenotes/notes/bump-go-1.20.7-0f9b7e391c8ab4b6.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
```

Comment:
Should this be under sec too?

Reply:
I'll add this in another PR since we haven't done it for the previous go bump (also I have a green CI so I'll merge and fix the changelog later !).